src/migration/1701100478531-PostRelationReplica.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class PostRelationReplica1701100478531 implements MigrationInterface {
```

Comment:
This should be together with the above migration :)

Reply:
The issue with migrations was because we thought multiple files could break production due to one migration failing or not. Then we said it does not matter because typeorm [runs them together in transaction](https://dailydotdev.slack.com/archives/C02E2C3C13R/p1697713738231189?thread_ts=1697711684.457159&cid=C02E2C3C13R) right? 

No need to bundle them when they do two separate things.

Reply:
CC: @idoshamun